### DISCUSSION TITLE
Dropdownlist from module to select from another module

### DISCUSSION THREAD

**Discussion by joshygeorge13** [oqtane/oqtane.framework#703] [General]
Hi, I am new here so pardon if my questions seem basic.
I have 3 modules named customers and products(courtesy to Mike Casas) and orders module. How can i make a dropdown list to select a customer (from list of customers in module) and the product from product module to order module?

**Comment by mikecasas** [comment]
Do you have three separate modules - three different solutions? Or do you have them all in one solution?

**Reply by joshygeorge13** [reply]
I have 3 seperate modules in one solution.

**Reply by joshygeorge13** [reply]
Please see the repository for the files which I made earlier with blazor for the functionality. Now I like to create the project using Oqtane.
https://github.com/joshygeorge13/Calms

**Reply by mikecasas** [reply]
Do you have an oqtane project yet? If not, check out this blog on how to create one. Once you have it set up, you can copy those files into the oqtane project.

https://blazorhelpwebsite.com/ViewBlogPost/4

**Reply by joshygeorge13** [reply]
@mikecasas  I have created a couple of projects on Oqtane for checking the functionality. The solution I am looking fo is to create internal or external modules which can be related as i mentioned in the files uplaoded to github repository. When I created the Oqtane solution and copied files from my existing projects it seems to work but the event loggers and audit doesn't seems to work.

**Reply by mikecasas** [reply]
Do you have a link to those oqtane projects?

**Reply by joshygeorge13** [reply]
@mikecasas The project was created as a test and I have removed them. Anyway I will redo the same and share it with you asap.
